src/index.ts

```diff
@@ -87,34 +88,59 @@ joplin.plugins.register({
 			name: "setDefaultNoteTemplate",
 			label: "Set default note template",
 			execute: async () => {
-				console.log("Command");
+				const templateId = await getUserTempateSelection(templatesFolderId, "id");
+				if (templateId) {
+					await joplin.settings.setValue("defaultNoteTemplateId", templateId);
+					await joplin.views.dialogs.showMessageBox("Default note template set successfully!");
+				}
 			}
 		});
 
         await joplin.commands.register({
 			name: "setDefaultTodoTemplate",
 			label: "Set default to-do template",
 			execute: async () => {
-				console.log("Command");
+				const templateId = await getUserTempateSelection(templatesFolderId, "id");
+				if (templateId) {
+					await joplin.settings.setValue("defaultTodoTemplateId", templateId);
+					await joplin.views.dialogs.showMessageBox("Default to-do template set successfully!");
+				}
```

Comment:
You might want to consider combining this and the above command for code re-usability sake. You could create a function that takes the setting name, and a dialog box message and call it from here and the default template command.

Reply:
I think we can do this later on. Because the code is repeated just two times and it's only 2 lines of code.

Reply:
Fair enough, you're the owner of this codebase so these calls are ultimately yours.